app/setup-tray-icon.js

```diff
@@ -0,0 +1,28 @@
+import path from 'path'
+import { app, Menu, Tray } from 'electron'
+
+let appIcon = null
+const iconPath = path.join(__dirname, '../docs/logo.png')
+
+function setupTrayIcon () {
```

Comment:
Disable here the trayicon on macOS! There is no need for that as the app on that platform will not quit if all the windows are closed.